runner/repositories_test.go

```diff
@@ -51,6 +55,25 @@ type RepoTestSuite struct {
 	Runner   *Runner
 }
 
+func (s *RepoTestSuite) reposMapValues(repos map[string]params.Repository) []params.Repository {
+	reposSlice := []params.Repository{}
+	for _, value := range repos {
+		reposSlice = append(reposSlice, value)
+	}
+	return reposSlice
+}
+
+func (s *RepoTestSuite) equalReposByName(expected, actual []params.Repository) {
+	s.Require().Equal(len(expected), len(actual))
+
+	sort.Slice(expected, func(i, j int) bool { return expected[i].Name > expected[j].Name })
+	sort.Slice(actual, func(i, j int) bool { return actual[i].Name > actual[j].Name })
+
+	for i := 0; i < len(expected); i++ {
```

Comment:
You also need to test that the lengths of both slices match. The ```expected``` array may be shorter, so you stop checking after you run out of elements. But in reality they are not equal.

Reply:
scratch that. You do check above. My mistake!